samples/client/petstore/java/jersey2-java8-localdatetime/src/main/java/org/openapitools/client/model/Cat.java

```diff
@@ -46,33 +38,6 @@
 })
 
 public class Cat extends Animal {
-  public static final String JSON_PROPERTY_DECLAWED = "declawed";
-  private Boolean declawed;
-
-
-  public Cat declawed(Boolean declawed) {
-    this.declawed = declawed;
-    return this;
-  }
-
-   /**
-   * Get declawed
-   * @return declawed
-  **/
-  @javax.annotation.Nullable
-  @ApiModelProperty(value = "")
-  @JsonProperty(JSON_PROPERTY_DECLAWED)
-  @JsonInclude(value = JsonInclude.Include.USE_DEFAULTS)
-
-  public Boolean getDeclawed() {
-    return declawed;
-  }
-
-
-  public void setDeclawed(Boolean declawed) {
-    this.declawed = declawed;
-  }
-
```

Comment:
@spacether the `declawed` property has been removed as part of the change, which doesn't seem to be correct according to the definition of `Cat`:
```
    Cat:
      allOf:
        - $ref: '#/components/schemas/Animal'
        - $ref: '#/components/schemas/Address'
        - type: object
          properties:
            declawed:
              type: boolean
```
ref: https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/test/resources/3_0/java/petstore-with-fake-endpoints-models-for-testing-with-http-signature.yaml#L1411-L1418

Reply:
That is correct as declawed only exists in that allOf third schema. It does not have an explicit definition inside Cat because cat has by default:
```
Cat:
  properties: {} # no declawed property here
  additionalProperties: true
```

Reply:
> The CodegenModel Cat not containing declawed when vars only contains properties in the current schema is correct

Can you share the link(s) to discussion supporting this claim so that everyone is on the same page?

Reply:
Yup it is in the link I pasted above https://github.com/OAI/OpenAPI-Specification/issues/2478